bin/src/worker.rs

```diff
@@ -12,12 +12,11 @@ use std::{
 
 #[cfg(target_os = "macos")]
 use libc::c_char;
-use libc::{self, pid_t};
+use libc::pid_t;
```

Comment:
Should it be prefix with `#[cfg(target_os = "...")]` ?

Reply:
`pid_t` is not bound to mac_os compilation